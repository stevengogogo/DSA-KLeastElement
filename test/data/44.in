10 50000
-4827 9678 4091 1652 -5039 3000 -4838 -3146 8483 6590
Delete 2
Reverse 9 9
Delete 9
Delete 2
Query 2 3 2
Delete 3
Delete 3
Delete 3
Delete 4
Insert 3 4087
Insert 4 3507
Delete 4
Delete 4
Delete 1
Reverse 1 1
Insert 2 155
Insert 2 409
Delete 2
Delete 3
Query 2 2 1
Query 1 1 1
Insert 3 -2406
Delete 1
Insert 3 2685
Query 2 3 2
Delete 1
Insert 1 3616
Query 3 3 1
Insert 3 281
Query 2 2 1
Insert 3 -6433
Query 5 5 1
Reverse 3 3
Reverse 3 5
Delete 5
Delete 3
Reverse 3 3
Query 2 3 1
Reverse 3 3
Delete 1
Query 2 2 1
Reverse 1 2
Delete 2
Query 1 1 1
Insert 1 4431
Delete 1
Reverse 1 1
Query 1 1 1
Query 1 1 1
Insert 1 -2199
Reverse 2 2
Query 2 2 1
Delete 2
Reverse 1 1
Delete 1
