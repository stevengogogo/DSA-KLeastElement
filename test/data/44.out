1652
155
1652
2685
2685
-2406
2685
-2406
281
281
281
281
281
